mintlify/mint
# Cli clone workaround

## Summary
For users that cloned the repo, the commands work slightly differently. This PR allows the CLI to detect when they're on an old version of `mint` and delete its content accordingly.

## Test Plan
1. Navigate to `~/.mintlify`
2. Delete `mint`
3. `git clone https://github.com/mintlify/mint.git --branch legacy-components-import`
4. Run a local version of this CLI.
5. Make sure it runs
